lessons_10/test_homeworks10.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+from homework_10 import factorial, is_prime, count_vowels, unique_elements, fibonacci, sum_numbers_in_list
+
+class TestHomeworks(unittest.TestCase):
+    def test_factorial(self):
+        self.assertEqual(factorial(5), 120)
+        self.assertEqual(factorial(7), 5040)
+        self.assertEqual(factorial(10), 3628800)
+        with self.assertRaises(ValueError):
+            factorial(-10)
+        self.assertEqual(factorial(1), 1)
+    
+    def test_is_prime(self):
+        self.assertTrue(is_prime(2))
+        self.assertTrue(is_prime(37))
+        self.assertFalse(is_prime(100))
+        self.assertFalse(is_prime(57))
+        self.assertTrue(is_prime(97))
+        self.assertFalse(is_prime(91))
+    
+    def test_count_vowels(self):
+        self.assertEqual(count_vowels("abracadabra"), 5)
+        self.assertEqual(count_vowels("ABCDEFGHIJKLMNOPQRSTUVWXYZ"), 5)
+        self.assertEqual(count_vowels("1234567890"), 0)
+        self.assertEqual(count_vowels("Why so serious?"), 5)
+        self.assertEqual(count_vowels("gym"), 0)
+    
+    def test_unique_elements(self):
+        self.assertEqual(set(unique_elements([1, 3, 3, 7, 8, 8, 9, 9, 9])), {1, 3, 7, 8, 9})
```

Comment:
```suggestion
        self.assertEqual(set(unique_elements([1, 3, 3, 7, 8, 8, 9, 9, 9])), {1, 3, 7, 8, 9})
```
тут щось дивне, усюди у вас ліст у повернені а тут сет....